sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -55,11 +55,13 @@
 @ServiceClient(builder = CryptographyClientBuilder.class, isAsync = true, serviceInterfaces = CryptographyService.class)
 public class CryptographyAsyncClient {
     static final String KEY_VAULT_SCOPE = "https://vault.azure.net/.default";
+    public static final String SECRETS_COLLECTION = "secrets";
```

Comment:
Your PR comment says there are no public API changes, but this is a new public API. Is it required?

Reply:
yeah, that's not needed.
made it private.